server/src/main/java/org/opensearch/index/search/stats/SearchStats.java

```diff
@@ -86,9 +86,9 @@ public long getTimeInMillis() {
 
         @Override
         public void writeTo(StreamOutput out) throws IOException {
-            out.writeVLong(current);
+            out.writeZLong(current);
             out.writeVLong(total);
-            out.writeVLong(timeInMillis);
+            out.writeZLong(timeInMillis);
```

Comment:
Is ZLong needed for timeInMillis considering it is never decremented?

Reply:
It can be incremented by a negative value. It should never occur and I have not seen it in practice but changed it to be safe. 
```
phaseStats.timing.inc(TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - context.getCurrentPhase().getStartTimeInNanos()));

```